modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -1,100 +1,82 @@
-as the file name for `clientCA` in the identity provider configuration.
++
+should proxy to
 +
 ----
-# oc adm ca create-signer-cert \
-  --cert='/etc/origin/master/proxyca.crt' \
-  --key='/etc/origin/master/proxyca.key' \
-  --name='openshift-proxy-signer@1432232228' \
-  --serial='/etc/origin/master/proxyca.serial.txt'
+\https://<namespace_route>/oauth/authorize?...
 ----
 +
+. Subpaths of the URL that proxies to `\https://<namespace_route>/oauth/authorize` must proxy to subpaths of `\https://<namespace_route>/oauth/authorize`. For example:
++
+----
+\https://proxy.example.com/login-proxy/authorize/approve?...
+----
++
+should proxy to
++
+----
+\https://<namespace_route>/oauth/authorize/approve?...
+----
+
 [NOTE]
 ====
-The `oc adm ca create-signer-cert` command generates a certificate that is valid
-for five years. This can be altered with the `--expire-days` option, but for
-security reasons, it is recommended to not make it greater than this
-value.
-
-// Run `oc adm` commands only from the first master listed in the Ansible host inventory file,
-// by default *_/etc/ansible/hosts_*.
+The `https://<namespace_route>/` address is the location where `.well-known`
+link:https://tools.ietf.org/html/rfc5785[RFC 5785] resources containing
+information about the authorization server are published.
 ====
 
-. Generate a client certificate for the proxy. You can generate this certificate
-by using any x509 certificate tooling, including the `oc adm` CLI:
-+
-----
-# oc adm create-api-client-config \
-  --certificate-authority='/etc/origin/master/proxyca.crt' \
-  --client-dir='/etc/origin/master/proxy' \
-  --signer-cert='/etc/origin/master/proxyca.crt' \
-  --signer-key='/etc/origin/master/proxyca.key' \
-  --signer-serial='/etc/origin/master/proxyca.serial.txt' \
-  --user='system:proxy' <1>
-
-# pushd /etc/origin/master
-# cp master.server.crt /etc/pki/tls/certs/localhost.crt <2>
-# cp master.server.key /etc/pki/tls/private/localhost.key
-# cp ca.crt /etc/pki/CA/certs/ca.crt
-# cat proxy/system\:proxy.crt \
-  proxy/system\:proxy.key > \
-  /etc/pki/tls/certs/authproxy.pem
-# popd
-----
-<1> The user name can be anything, however it is useful to give it a descriptive
-name as it will appear in logs.
-<2> When running the authentication proxy on a different host name than the
+.Prerequisites
+
+* Obtain the `mod_auth_gssapi` module from the
+link:https://access.redhat.com/solutions/392003[Optional channel].
+You must have following packages installed on your local machine:
+** `httpd`
+** `mod_ssl`
+** `mod_session`
+** `apr-util-openssl`
+** `mod_auth_gssapi`
+
+* Generate a CA for validating requests that submit the trusted header. Use this CA
+as the file name for `clientCA` in the identity provider configuration.
+
+* Generate a client certificate for the proxy. You can generate this certificate
+by using any x509 certificate tooling.
+
+[NOTE]
+====
+When running the authentication proxy on a different host name other than
```

Comment:
You can omit this note, it refers quite a lot to the `oc adm create-api-client` command, in most cases the proxy will not be running on the master anyway.